### PR TITLE
Limit targest o 10 per user

### PR DESCRIPTION
**Pull request by peter7z**
Adds a maximum of 10 targets per user, and returns status bad_request when the user has 10 already.

### CODE REVIEW

File: app/models/target.rb [hunk 1]
```diff
@@ -25,4 +25,12 @@ class Target < ApplicationRecord
   validates :title, presence: true
   validates :radius, presence: true, numericality: { greater_than: 0, less_or_equal_than: 5000 }
   validates :latitude, :longitude, presence: true, numericality: true
+
+  validate :targets_count_limit, on: :create
+
+  def targets_count_limit
+    return unless self.user.targets.size >= 10
+
+    errors.add(:limit, 'Limit exceeded, 10 targets maximum')
```
Comment (jpascual1994) [inline app/models/target.rb]: Are you using I18n? Maybe you can move it to the translation file
https://guides.rubyonrails.org/i18n.html

File: spec/requests/targets/create_spec.rb [hunk 2]
```diff
@@ -47,4 +47,14 @@
     end
   end
 
+  context 'when the user has 10 targets already' do
+    let!(:targets) { create_list(:target, 10, user: user) }
+
```
Comment (jpascual1994) [inline spec/requests/targets/create_spec.rb:51]: Remove extra line :)

**Review by gabrielbursztein2 — APPROVED** [review]
Approve after pascui's comments.

**Review by TimoPeraza — APPROVED** [review]
Well done 🥇

**Pull request merged**